shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/DashboardUserMapper.java

```diff
@@ -74,6 +74,14 @@ public interface DashboardUserMapper extends ExistProvider {
      */
     List<DashboardUserDO> selectByQuery(DashboardUserQuery dashboardUserQuery);
     
+    /**
+     * select by ids.
+     *
+     * @param ids ids.
+     * @return users
+     */
+    List<DashboardUserDO> selectByIds(@Param("ids") List<String> ids);
```

Comment:
better change to `List<DashboardUserDO> selectByIds(@Param("ids") Set<String> ids);`